content/blog/policy-management-policy-packs/index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Guardrails for Neo: New Policy Management Experience"
+date: 2025-09-17
+authors:
+    - luke-ward
+    - dan-biwer
+    - insights-team
+meta_desc: "Announcing streamlined user experience for Pulumi Policy and new pre-built policy packs, providing the essential guardrails for AI automation with Pulumi Neo."
```

Comment:
Not just AI automation: for all cloud infrastructure automation, whether AI, developer self-serve, or otherwise.